packages/react/src/components/OverflowMenu/next/index.js

```diff
@@ -99,6 +120,17 @@ OverflowMenu.propTypes = {
    */
   label: PropTypes.string,
 
+  /**
+   * Specify how the menu should align with the button element
+   */
+  menuAlignment: PropTypes.oneOf([
+    'top-left',
```

Comment:
Should this variant also allow `top` and `bottom`?

Reply:
no not for this one